haas/client/project.py

```diff
@@ -0,0 +1,43 @@
+from haas.client.base import *
+from haas.client.client_errors import *
+
+
+class Project(ClientBase):
+        """ Consists of calls to query and manipulate project related
+            objects and relations """
+
+        def list(self):
+            """ Lists all projects under HIL """
+            url = self.object_url('/projects')
+            q = requests.get(url, headers={"Authorization": "Basic %s" % self.auth})
+            if q.ok:
+                return q.json()
+            elif q.status_code == 401:
+                raise AuthenticationError("Make sure credentials match chosen authentication backend.")
+
+        def nodes_in(self, project_name):
+            """ Lists nodes allocated to project <project_name> """
+            self.project_name = project_name
```

Comment:
Why assign this locally? Will future calls of the sort `Project.operation()` assume a built-in project?

If that's the case, maybe have an `__init__()` that binds the object to a particular class, then make this argument optional except when someone's calling the class vs an instantiated object?